cmd/oras/root/repo/tags.go

```diff
@@ -95,8 +100,25 @@ func showTags(cmd *cobra.Command, opts *showTagsOptions) error {
 	if err != nil {
 		return err
 	}
+
+	// if a repository path is given, filter the tags under the repository
+	var repoFilter func(ref string) (string, bool)
+	if opts.Target.Type == option.TargetTypeOCILayout {
+		ref, err := registry.ParseReference(opts.Reference)
```

Comment:
Should this be limited to filtering only tags that are in the format of registry/repo? Why ton `HasPrefix` or `Contains` on any string?

Reply:
We decide to make it limited to filtering only tags that are in the format of registry/repo, as this is the common use case we see from the users.